.github/workflows/node.yml

```
@@ -0,0 +1,27 @@
+name: Problem CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

We should have two workflows, on pushing to main for releasing and for PR to testing:

```suggestion
name: PR testing

on:
  pull_request:
    types: [opened, reopened, synchronize, ready_for_review]
```